PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Quality Assurance Checklist 
+To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 
+
+## Requirements 
+- [ ] Followed Coding Style Guide 
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it. 
+- [ ] Test provides useful information and uses relevant data to accurately represent BPS 
+    - NOTE: If test file already exists, use that one 
+- [ ] 2 Members of the team have already reviewed my PR 
+    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
```

Comment:
while we're at it, it's probably time to remove chase from the reviewers list in the readme